python_modules/dagster/dagster/_daemon/backfill.py

```diff
@@ -146,51 +147,68 @@ def execute_backfill_jobs(
 
         # refetch, in case the backfill was updated in the meantime
         backfill = cast(PartitionBackfill, instance.get_backfill(backfill_id))
-        # create a logger that will always include the backfill_id as an `extra`
-        backfill_logger = cast(
-            logging.Logger,
-            logging.LoggerAdapter(logger, extra={"backfill_id": backfill.backfill_id}),
-        )
-        try:
-            evaluation_time = pendulum.now("UTC")
-            tick = instance.create_tick(
-                TickData(
-                    instigator_origin_id=_BACKFILL_ORIGIN_ID,
-                    instigator_name=backfill_id,
-                    instigator_type=InstigatorType.BACKFILL,
-                    status=TickStatus.STARTED,
-                    timestamp=evaluation_time.timestamp(),
-                    selector_id=backfill_id,
-                )
+        evaluation_time = pendulum.now("UTC")
+
+        # TODO - probably need to scope by code location. for now just put everything together for testing
+        log_key = ["backfill", backfill_id, evaluation_time.strftime("%Y%m%d_%H%M%S")]
+        with InstigationLogger(
```

Comment:
From my understanding, the `InstigationLogger` has only been used for things like sensor logs where the logs are user defined. Since we want to have our backfill daemon logs appear in the UI, that could change some of the assumptions made by the `InstigationLogger` 

1. the `InstigationLogger` always uses the `dagster` logger, whereas in this case we want the backfill daemon logger https://github.com/dagster-io/dagster/blob/97c80ef86a5c077eafe007d0af87d7751e08a604/python_modules/dagster/dagster/_core/definitions/instigation_logger.py#L92